Makefile

```diff
@@ -48,8 +48,10 @@ install: Makefile.inc hwdata.pc
 	for foo in $(IDFILES) ; do \
 		install -m 644 $$foo $(DESTDIR)$(datadir)/$(NAME) ;\
 	done
-	mkdir -p -m 755 $(DESTDIR)$(libdir)/modprobe.d
-	install -m 644 -T blacklist.conf $(DESTDIR)$(libdir)/modprobe.d/dist-blacklist.conf
+	@if [ "$(blacklist)" == true ]; then \
```

Comment:
Ah, another nit: `==` is a bash extension, `=` is POSIX sh.

Reply:
Fixed, thanks!